server/graphql/programs/src/mutations/vaccination/insert.rs

```diff
@@ -106,7 +106,6 @@ fn map_error(error: ServiceError) -> Result<InsertVaccinationResponse> {
         | ServiceError::ProgramEnrolmentDoesNotMatchVaccineCourse
         | ServiceError::VaccinationAlreadyExistsForDose
         | ServiceError::ReasonNotProvided
-        | ServiceError::StockLineNotProvided
```

Comment:
Removes this error - there are so many cases now where we allow a stock line to not be provided/don't want to generate an invoice. Easier to let the caller pass in stock line if they care, otherwise don't worry about it, than validate every case.

The stockline part is not the core function (we want to record there was a vaccination, the stock transactions are helpful additions) so I think best not to block the vaccination being recorded